src/components/Cart/CartItem/CartItem.jsx

```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types';
+import * as Styled from './style';
+import { useEffect, useState } from 'react';
+import { parsePrice } from 'utils';
+import { MESSAGE } from 'constants';
+import smallTrashBin from 'assets/svg/smallTrashbin.svg';
+import CheckBox from 'components/Common/CheckBox/CheckBox';
+import useCart from 'hooks/useCart';
+
+const CartItem = ({
+  id,
+  imgUrl,
+  name,
+  price,
+  quantity,
+  isSelected,
+  onToggleSelect,
+}) => {
+  const { updateItemQuantity, deleteItem } = useCart();
+  const [itemQuantity, setItemQuantity] = useState(quantity);
+
+  const handleIncrementQuantity = () => {
+    setItemQuantity((prevQuantity) => prevQuantity + 1);
+  };
+
+  const handleDecrementQuantity = () => {
+    try {
+      if (itemQuantity === 1) throw new Error(MESSAGE.MINIMUM_CART_LENGTH);
+      setItemQuantity((prevQuantity) => prevQuantity - 1);
+    } catch (error) {
+      alert(error.message);
+    }
+  };
+
+  const handleDeleteItem = () => {
+    deleteItem(id);
+  };
+
+  useEffect(() => {
+    if (quantity !== itemQuantity) {
+      updateItemQuantity(id, itemQuantity);
+    }
+  }, [itemQuantity]);
```

Comment:
```suggestion
   const { quantity, decrease, increase, drop } = useCartItemQuantity();
```
요러한 인터페이스를 사용하는 custom hook은 어떠실까용?
잡아드린 코드라인에 대한 부분을 코드를 보는 입장에서 구체적으로 알 필요가 있을까요?